doc/api/deprecations.md

```diff
@@ -2518,6 +2518,26 @@ Type: Documentation-only
 Prefer [`response.socket`][] over [`response.connection`] and
 [`request.socket`][] over [`request.connection`].
 
+<a id="DEP0XXX"></a>
+### DEP0XXX: http/2 request.finished and response.finished
+<!-- YAML
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/28679
+    description: Documentation-only deprecation.
+-->
+
+Type: Documentation-only
+
+[`response.finished`][] indicates whether [`response.end()`] has been
```

Comment:
I think the linter is flagging this:

```suggestion
[`response.finished`][] indicates whether [`response.end()`][] has been
```

Reply:
(I think the reference might also need to be added.)

Reply:
Oh, so the line numbers on the linter are not reliable? Or maybe I'm misreading.